src/components/UniversalTable/UniversalTable.module.scss

```diff
@@ -28,7 +28,7 @@
     padding: 10px 20px;
     display: grid;
     grid-template-columns: repeat(auto-fit, minmax(100px, 1fr));
-    justify-content: space-around;
+    justify-content: space-between;
```

Comment:
why are yo so sure it won't brake anything on other pages that use the same component?

Reply:
no matter how many columns it will be. First colum shoul be lies on left side of the table, least one should be lying on rigrt side. And other space shoul be divide into equal parts.